lib/util/BandUtil.js

```diff
@@ -97,6 +97,52 @@ export function resetAllBands(activity, diBands, bounds) {
   }
 }
 
+/**
+ * Updates all attributes concerning size in shape and DI of the bands according to their
+ * activities size.
+ * Does not use the event bus, so changes are not immediately displayed.
+ *
+ * @param {Object} activityShape shape object of the choreography activity
+ */
+export function updateBandsBounds(activityShape) {
```

Comment:
Seems to have a lot of code in common with `updateBounds`, maybe there's potential to merge them? I see that it might make them unnecessarily complex, though.

Reply:
No longer in use. Will be removed